mappings/services.json

```diff
@@ -652,161 +670,197 @@
                 "bpss"
               ]
             }
+          },
+          {
+            "hash_to": {
+              "field": "id",
+              "target_field": "serviceIdHash"
+            }
           }
         ]
       },
       "dynamic": "strict",
       "properties": {
-        "id": {
+        "dmtext_id": {
           "type": "keyword"
         },
-        "service_id_hash": {
+        "dmsortonly_serviceIdHash": {
           "type": "keyword",
           "include_in_all": false
         },
-        "lot": {
-          "type": "keyword",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
+        "dmagg_lot": {
+          "type": "keyword"
+        },
+        "dmtext_lot": {
```

Comment:
If the mapping is `keyword`, why use the `dmtext_` prefix over `dmfilter_`?

Reply:
Just trying to be bug-compatible with the previous mapping. Should be the direct equivalent.

Reply:
(i think this is for cases we explicitly don't want any stemming)